Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:trusty
+
+# Get the basic VIC dependencies
+RUN apt-get update -q && \
+    apt-get install -y -q --no-install-recommends \
+        ca-certificates \
+        build-essential \
+        ssh \
+        netcdf-bin \
+        libnetcdf-dev \
+        libopenmpi-dev \
+        openmpi-bin \
+        git \
+        make
+
+# Dedicated work directory for output
+RUN mkdir $HOME/workdir
```

Comment:
I'll export it as a variable as well.  This was mostly so that we have a clear directory to write output to when we run our test setups.